app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  validates :email, uniqueness: true, presence: true
+  validates_presence_of :password, require: true
+
+  has_secure_password
```

Comment:
I learned this yesterday, but the has_secure_password does the same thing as validates_presence_of :password